bridge/opencensus/internal/ocmetric/metric.go

```diff
@@ -152,22 +156,93 @@ func convertHistogram(labelKeys []ocmetricdata.LabelKey, ts []*ocmetricdata.Time
 				Sum:          dist.Sum,
 				Bounds:       dist.BucketOptions.Bounds,
 				BucketCounts: bucketCounts,
+				Exemplars:    exemplars,
 			})
 		}
 	}
 	return metricdata.Histogram[float64]{DataPoints: points, Temporality: metricdata.CumulativeTemporality}, err
 }
 
-// convertBucketCounts converts from OpenCensus bucket counts to slice of uint64.
-func convertBucketCounts(buckets []ocmetricdata.Bucket) ([]uint64, error) {
+// convertBuckets converts from OpenCensus bucket counts to slice of uint64,
+// and converts OpenCensus exemplars to OpenTelemetry exemplars.
+func convertBuckets(buckets []ocmetricdata.Bucket) ([]uint64, []metricdata.Exemplar[float64], error) {
 	bucketCounts := make([]uint64, len(buckets))
+	exemplars := []metricdata.Exemplar[float64]{}
+	var err error
 	for i, bucket := range buckets {
 		if bucket.Count < 0 {
-			return nil, fmt.Errorf("%w: %q", errNegativeBucketCount, bucket.Count)
+			err = errors.Join(err, fmt.Errorf("%w: %q", errNegativeBucketCount, bucket.Count))
+		} else {
+			bucketCounts[i] = uint64(bucket.Count)
+		}
+		if bucket.Exemplar != nil {
+			exemplar, exemplarErr := convertExemplar(bucket.Exemplar)
+			if exemplarErr != nil {
+				err = errors.Join(err, exemplarErr)
+			} else {
+				exemplars = append(exemplars, exemplar)
+			}
+		}
```

Comment:
I think that the following would be more idiomatic (mostly https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow).

I also think that converting exemplars is not needed when `bucket.Count < 0` as the caller ignores the result if there is an error anyway.

```suggestion
			err = errors.Join(err, fmt.Errorf("%w: %q", errNegativeBucketCount, bucket.Count))
			continue
		}
		bucketCounts[i] = uint64(bucket.Count)

		if bucket.Exemplar != nil {
			exemplar, exemplarErr := convertExemplar(bucket.Exemplar)
			if exemplarErr != nil {
				err = errors.Join(err, exemplarErr)
				continue
			}
			exemplars = append(exemplars, exemplar)
		}
```